tasks/4.test.js

```diff
@@ -0,0 +1,13 @@
+const { removeDuplicates } = require("./4");
+
+describe("Task #4", () => {
+  describe("removeDuplicates()", () => {
+    it("should remove duplicates in an array", () => {
+      const array = ["one", "two", "three", "one", "four", "five", "six"];
+      const result = removeDuplicates(array);
```

Comment:
Your function does not accept any arguments, it operates on it's own constant of array.